### ISSUE TITLE
[$250] Other A11y: Global: The page title does not describe the purpose of page

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#74767]
## Action Performed

Prerequisite:

* User is logged in

1. Using Chrome, open the URL: [https://new.expensify.com/](<https://new.expensify.com/>)
2. Activate the "My Settings" button
3. Activate the "Subscription" menu item
4. Inspect the code and note the title of the page.

## Expected Result

The page title should clearly describe the purpose or content of the page to help users understand its context and differentiate it from other open tabs, e.g.:

* 'Settings - Subscription - Expensify' or similar

## Actual Result

Currently, the page title is displayed as “(1) New Expensify,” which does not provide a meaningful or descriptive indication of the page’s purpose or content. As a result, users may have difficulty identifying the page when multiple tabs are open.

HTML:

(1) New Expensify

## Area issue was found

Settings–Subscription

## Failed Checkpoint

WCAG 2.4.2 Page Titled<br>[https://www.w3.org/WAI/WCAG22/Understanding/page-titled.html](<https://www.w3.org/WAI/WCAG22/Understanding/page-titled.html>)<br>[https://www.w3.org/WAI/WCAG22/Understanding/page-titled.html](<https://www.w3.org/WAI/WCAG22/Understanding/page-titled.html>)

## User Impact

People with visual disabilities will benefit from being able to differentiate content when multiple Web pages are open. People with cognitive disabilities, limited short-term memory and reading disabilities also benefit from the ability to identify content by its title. People with severe mobility impairments whose mode of operation relies on audio when navigating between Web pages will benefit from page titles.

## Suggested Resolution

Update the element to include clear and descriptive text that reflects the specific page content or function, rather than a generic label, e.g.:

* 'Settings - Subscription - Expensify' or similar<br>[https://www.w3.org/WAI/WCAG22/Understanding/page-titled.html](<https://www.w3.org/WAI/WCAG22/Understanding/page-titled.html>)<br>[https://www.w3.org/WAI/WCAG22/Understanding/page-titled.html](<https://www.w3.org/WAI/WCAG22/Understanding/page-titled.html>)

## Environment(s)

Windows 11 Home - Chrome

<img src="https://github.com/user-attachments/assets/d74920da-f2cb-4eba-b0a0-076150edd214 " alt="Image" width="1918" data-linear-height="1020" />

**Version Number:** NA<br>**Reproducible in staging?:** NA<br>**Reproducible in production?:** Yes<br>**Email or phone of affected tester (no customers):** NA<br>**Logs:** [https://stackoverflow.com/c/expensify/questions/4856](<https://stackoverflow.com/c/expensify/questions/4856>) NA<br>**Notes/Photos/Videos:** Any additional supporting documentation Yes<br>**Expensify/Expensify Issue URL:** [https://platform.applause.com/products/36605/community-issues/6995830](<https://platform.applause.com/products/36605/community-issues/6995830>)<br>**Issue reported by:** NA<br>**Slack conversation:** NA<br>**Reproducible in staging?:** NA<br>**Reproducible in production?:** Yes<br>**Email or phone of affected tester (no customers):** NA<br>**Logs:** [https://stackoverflow.com/c/expensify/questions/4856](<https://stackoverflow.com/c/expensify/questions/4856>) NA<br>**Notes/Photos/Videos:** Any additional supporting documentation Yes<br>**Expensify/Expensify Issue URL:** [https://platform.applause.com/products/36605/community-issues/6995830](<https://platform.applause.com/products/36605/community-issues/6995830>)<br>**Issue reported by:** NA<br>**Slack conversation:** NA

<details><summary>Upwork Automation - Do Not Edit</summary>

* Upwork Job URL: [https://www.upwork.com/jobs/\~022009685648638422846](<https://www.upwork.com/jobs/~022009685648638422846>)
* Upwork Job ID: 2009685648638422846
* Last Price Increase: 2026-01-28
* Automatic offers:
* daledah | Contributor | 110293722

</details>

<details><summary>Issue Owner</summary>

Current Issue Owner: @mallenexpensify

</details>

**Comment by mallenexpensify** [comment]
`Accessibility` issue, will be addressed soon.

**Comment by mananjadhav** [comment]
@daledah Can you check the previous comment.

**Comment by daledah** [comment]
friendly bump here @mananjadhav

**Comment by daledah** [comment]
@yuwenmemon waiting for the assignment [here](https://github.com/Expensify/App/issues/74767#issuecomment-3813006404)

**Comment by mallenexpensify** [comment]
@rushatgabhane , 👀 above plz, thx

**Comment by mananjadhav** [comment]
Thanks for the confirmation @yuwenmemon.

**Comment by JmillsExpensify** [comment]
Sorry about that. I agree with Yuwen on this comment

> This is a good question. I think the unread count should still show at the very beginning. So maybe (N) Page Title - New Expensify or simply just (N) Page Title. The fact that you're on Expensify is implied by the icon. Even just looking at the example from GitHub in your screenshots, this seems to be a pattern that other websites use.

**Comment by daledah** [comment]
thanks @JmillsExpensify, i'll update my PR soon

**Comment by yuwenmemon** [comment]
No payment due just yet. We're still working on a few kinks.

**Comment by yuwenmemon** [comment]
This is just waiting payment now

**Comment by rushatgabhane** [comment]
> Also.. is payment just for the above PR or are there more PRs

The bug is fixed. We have a [follow up cleanup issue](https://github.com/Expensify/App/issues/86679). We don't need to wait on it

~Hold on, what is happening here. The bug is fixed, right?~

**Comment by mananjadhav** [comment]
Yes that looks correct. I'll check the open issue in a while.

**Comment by twisterdotcom** [comment]
@mananjadhav has submitted payment for this report on ND [here](https://staging.new.expensify.com/search?q=sortBy:date%20sortOrder:desc%20type:expense%20groupBy:reports%20reportID:R00HWkcTKVPJ%20policyID:9132779747441AFB).

SO: https://stackoverflowteams.com/c/expensify/questions/7582
- [x] Vendor found in [VML](https://docs.google.com/spreadsheets/d/1Fs-umMmHhCCMzxiczyNydl9Rd6jdO0pQskYA3RvNWQc/edit?gid=1740618654#gid=1740618654): True. Status: Active
- [x] Amount requested: $250.00 (Matches the payment summary here: https://github.com/Expensify/App/issues/74767#issuecomment-4195183361.)
- [x] I, @twisterdotcom am not already the BZ assignee. I have approved this payment.

<details><summary>Zapier Logs</summary>
<b>Run ID:</b> <a href="https://zapier.com/editor/311565481/run/00040eee-0e19-ae55-b55a-820d4351ba64/run-details">00040eee-0e19-ae55-b55a-820d4351ba64</a>
</details>

**Comment by mallenexpensify** [comment]
I raised the price on the open issue since @mananjadhav mentioned multiple proposals didn't work
- [[$500] Web - Document title flickers when navigating away from expense report](https://github.com/Expensify/App/issues/86679#top)
Bumping to weekly

**Comment by mallenexpensify** [comment]
[Cleanup issue has a PR raised ](https://github.com/Expensify/App/issues/86679#issuecomment-4341600777)so gonna close this. Comment/reopen, if needed.

**Comment by quinthar** [comment]
Approved $250 for @daledah
